tests/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing.UnitTests/CompilerErrorTests.cs

```diff
@@ -72,6 +72,29 @@ class TestClass {
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task TestCSharpReorderedExplicitCompilerErrorWithExplicitInterfaceSymbol()
```

Comment:
📝 It took a while but I made sure to create a test that fails without this change